ZenPacks/zenoss/ZenPackLib/tests/test_zenpacklib_version.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+"""Test that this version of ZenPackLib displays correct version"""
+
+import os
+from ZenPacks.zenoss.ZenPackLib.tests.ZPLTestBase import ZPLTestBase
+from ZenPacks.zenoss.ZenPackLib import zenpacklib
+
+
+class TestInstalledZenPackLibVersion(ZPLTestBase):
+    """Test this version of ZenPackLib against relevant installed ZenPacks"""
+
+    def get_version(self):
+        '''extract the ZenPack version from setup.py'''
+        path = os.path.join(os.path.dirname(__file__), '..', '..', '..', '..', 'setup.py')
```

Comment:
This seems like it would fail in the product-assembly tests. There's no setup.py in that case because the ZenPack is installed as an egg, not in link mode.